$OpenBSD: patch-redis_c,v 1.3 2010/11/17 08:44:54 dcoppa Exp $

Fix an alignment problem that was causing a SIGBUS on strict alignment 
archs like SPARC

--- redis.c.orig	Sat Nov  6 10:09:28 2010
+++ redis.c	Wed Nov 17 08:35:50 2010
@@ -7447,7 +7447,7 @@ static int deleteIfVolatile(redisDb *db, robj *key) {
 
 static void expireGenericCommand(redisClient *c, robj *key, robj *param, long offset) {
     dictEntry *de;
-    time_t seconds;
+    long seconds;
 
     if (getLongFromObjectOrReply(c, param, &seconds, NULL) != REDIS_OK) return;
 
